example/ck_tile/03_gemm/gemm_basic.hpp

```diff
@@ -13,6 +13,7 @@
 
 #define CK_TILE_PIPELINE_COMPUTE 1
```

Comment:
Let's align those names with actuall pipeline name.
```suggestion
#define CK_TILE_PIPELINE_COMPUTE_V3 1
```